Module 1: Form validation

boniface193 opened the pull request:
###  validate contact form
 - When the user submits the form, I checked if the email is in lowercase.
 - If the validation is not OK, I threw an error message to the user near the submit button informing them of the error, and the form is not sent.

deboik193 committed:
project single page implemented

deboik193 committed:
popup modal with object items

nxnom committed:
Fix linters

deboik193 committed:
marge conflict solved

deboik193 committed:
form validation included in the js file

deboik193 committed:
linter error resolved

King-Benx committed:
fix linting errors

iambenkis commented on js/main.js:
```diff
@@ -0,0 +1,51 @@
+  const allInput = Array.from(formValidate.querySelectorAll('input'));
+  const isLowerCase = allInput[1];
+
+  if (isLowerCase.value !== isLowerCase.value.toLowerCase()) {
+    const errorMsg = document.querySelector('.error');
+    errorMsg.innerText = 'Please make sure your email address is in lower case';
+    errorMsg.classList.toggle('d-none');
+    params.preventDefault();
+    return false;
+  }
```
- [ ] Good job :clap: , keep doing the great work :+1:

boniface193 committed:
README.md changes

###Code review change implemented

 - The Key Features Section.
 - The MIT link in the file leads MIT.md

iambenkis commented on index.html:
```diff
@@ -241,7 +143,8 @@ <h3 class="nav-sub-sub-heading weight--500 text--info mt-12">
+                            class="contact-form mt-24">
                             <label for="" class="d-none">Full name</label>
                             <input type="text" maxlength="30" required name="fullName" placeholder="your full name"
                                 class="weight--400">
```
- [ ] [**OPTIONAL**] Kindly make sure you correct this title, because your portfolio will probably be viewed by many people :+1: 

![Screenshot from 2022-12-23 15-25-53 (1)](https://user-images.githubusercontent.com/80458199/209348440-00cb4838-8256-4cd1-a0f1-8ebfcc5a28db.png)

iambenkis reviewed (approved):
Hi @boniface193 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![ ](https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif)
## Highlights
Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

deboik193 committed:
MIT license added

deboik193 committed:
README file updated

deboik193 committed:
Merge branch 'detail-popup' of https://github.com/boniface193/my-portfolio into form-validation

deboik193 committed:
readme file was updated
